x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -68,7 +93,7 @@ fragment DIGIT
     ;
 
 fragment LETTER
-    : [A-Za-z]
+    : [a-z]
```

Comment:
The file is case-insensitive, no need to define upper case (doing that throws an error which is a nice benefit).